examples/quickstart.ipynb

```diff
@@ -0,0 +1,230 @@
+{
```

Comment:
<p>May be add link</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1343/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>I don't really know how to do it properly. Because we could have multiple verision of the library in documentation and we should select here only one link to one particular version.</p><p> </p><p>I tried to use this: https://myst-parser.readthedocs.io/en/latest/syntax/cross-referencing.html#reference-roles , but it doesn't work with notebooks converted by nbsphinx. Probably, MyST-nb could work with this, but it requires changing the current extension for compiling notebooks.</p><p> </p><p>I've found that: https://docs.readthedocs.io/en/stable/guides/jupyter.html</p>

Reply:
<p>Not sure it is critical</p>